src/containers/BooksList.js

```diff
@@ -1,30 +1,47 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable array-callback-return */
 /* eslint-disable react/forbid-prop-types */
```

Comment:
With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.